store/queries/post_hotness.sql

```diff
@@ -0,0 +1,24 @@
+-- name: DeleteOldPostHotness :execrows
+DELETE FROM post_hotness
+WHERE generated_at < NOW() - sqlc.arg(retention_period)::INTERVAL;
+
+-- name: MaterializeClassicPostHotness :one
+WITH seq AS (SELECT NEXTVAL('post_hotness_generation_seq') AS seq)
+
+INSERT INTO post_hotness (uri, alg, score, generation_seq)
+SELECT
+    cp.uri AS uri,
+    'classic' AS alg,
+    (
+        SELECT COUNT(*)
+        FROM candidate_likes AS cl
+        WHERE cl.subject_uri = cp.uri AND cl.deleted_at IS NULL
+    )
+    / (EXTRACT(EPOCH FROM NOW() - cp.created_at) / (60 * 60) + 2)
```

Comment:
We need to make this clearer - maybe break it over more lines and add some more brackets and possibly even some inline comments 😓